bin/relation-graph

```diff
@@ -0,0 +1 @@
+../relation-graph/bin/relation-graph
```

Comment:
not sure about this

Reply:
...weird..I do not see this line of code in my IDE.